pallets/credit/src/benchmarking.rs

```diff
@@ -25,7 +25,7 @@ use frame_support::traits::Currency;
 use frame_system::RawOrigin;
 use node_primitives::{
     credit::{CreditData, CreditLevel, CreditSetting},
-    user_privileges::Privilege,
+    user_privileges::PrivilegeMapping,
```

Comment:
What's the difference between `Privilege` and `PrivilegeMapping`? I think they are the same.

Reply:
PrivilegeMapping is used placing Privilege in function params  to avoid some required trait ,Encode,Decode etc.